src/transformers/models/wav2vec2/modeling_wav2vec2.py

```diff
@@ -669,6 +667,256 @@ def custom_forward(*inputs):
         )
 
 
+class GumbelVectorQuantizer(nn.Module):
+    def __init__(
+        self,
+        dim,
+        num_vars,
+        temp,
+        groups,
+        combine_groups,
+        vq_dim,
+        time_first,
+        activation=nn.GELU(),
```

Comment:
if this function isn't really changed - I think we can hardcode it to `nn.GELU()` inside the class

Reply:
Don't think we need to allow for too many possibilities (we can always add more options later - however we cannot delete any options anymore due to breaking backwards compatibility)

Reply:
I've removed the unused params and some logic that the auhors didn't end up using (e.g. combining codebooks, multilayer vector projection). 
https://github.com/huggingface/transformers/pull/11306/files#diff-fde96a141d70737bff942cb61341f3b4b87729c9a066ecee4bfc86dfe590a8e6R700